test/unit/afdatetime/test_timezones.py

```diff
@@ -22,3 +23,19 @@ def test_pst(self):
             datetime.datetime(2015, 8, 1, 10, 9, 8))
         expected = (datetime.timedelta(hours=-7), '-07:00')
         assert expected == actual
+
+class TestDstAccurateTimeZone(object):
```

Comment:
thanks for adding adding unit tests!

Reply:
might be helpful to have a failure case - i.e. invalid timezone strings, invalid lat/lng.